src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Debug/ConsoleDebugLaunchProviderTests.cs

```diff
@@ -377,7 +377,7 @@ public void GetExeAndArgumentsWithEmptyArgs()
         }
 
         [Fact]
-        public async Task QueryDebugTargetsAsync_WhenLibraryWithoutRunCommand_ReturnsTargetPath()
+        public async Task QueryDebugTargetsAsync_WhenLibraryWithoutRunCommand_ReturnsEmptyExecutablePath()
```

Comment:
This will break unit testing debugging, this was a very deliberate change in https://github.com/dotnet/project-system/pull/3648.